### ISSUE TITLE
SafeDeleteQuerySet does not have a sorted property

### ISSUE THREAD

**Issue by bverhoeve** [makinacorpus/django-safedelete#155]
Hi! Thanks for the work on `django-safedelete` it's been working well on my current project. Django defines a `sorted` property on its `QuerySet` object which indicates if an `order_by` was called on the `QuerySet`.

The `SafeDeleteQuerySet` doesn't seem to have this property. Not a dealbreaker per se, but I think it's useful that `SafeDeleteQuerySet` supports as much as possible the same API as `QuerySet`. 

This should be quite straightforward to implement, so happy to help out on this issue.

**Comment by Gagaro** [comment]
I agree that we should be as close as possible to the QuerySet API, feel free to do a PR :+1: .
